src/_includes/common-form-fields.html

```diff
@@ -0,0 +1,134 @@
+{# ==========================================================================
+
+   common_form_fields.state()
+
+   ==========================================================================
+
+   Description:
+
+   Returns a select box with all 50 states and DC.
+
+   legend: Name of the field.
+
+   size: The size of the fields. Defaults to 1-3.
+          Possible options: 1, 1-2, 1-3
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro state(size='1-3', required=false, disabled=false) %}
+  {% from 'macros/form.html' import select as select %}
+  {{ select('State',
+            'Select state',
+            [ 'Alabama',
```

Comment:
You're going to want a key to go with the display text because converting spaces on the fly always leads to trouble. Not sure what our backend will be using (will there ever be queries based on state? Like for events?), but I've always gone by the abbreviation because that seems most common. Ex https://gist.github.com/jimmynotjim/8446233